web/book/src/tutorial/annotated_example.md

```diff
@@ -0,0 +1,39 @@
+# Annotated example
```

Comment:
We could unify this example with the example in the readme, and move it to the front page of the book ( #3173)